packages/string-templates/package.json

```diff
@@ -12,7 +12,9 @@
       "import": "./dist/bundle.mjs"
     },
     "./package.json": "./package.json",
-    "./test/utils": "./test/utils.js"
+    "./test/utils": "./test/utils.js",
+    "./iife": "./src/iife.js",
+    "./snippet": "./src/helpers/snippet.js"
```

Comment:
I'm guessing this is for testing purposes because it appears to be the `marked` library again. I would name this more descriptively so future people have a better idea what it's for.

Reply:
The snippet export is a testing leftover yes, so I'll remove that one. The `iife` export is deliberate and required - do you mean rename that to be more descriptive?